File: Jack.Business/Representante.vb
Public Class Representante

    Public Sub New()

    End Sub
    Public Function ObterMaes(intFamilia As Integer) As List(Of Model.Familia)

        Dim lstDados As List(Of Model.Familia) = Nothing
        Dim objDados As Data.Representante = Nothing

        Try

            objDados = New Data.Representante
            lstDados = New List(Of Model.Familia)
            lstDados = objDados.ObterMaes(intFamilia)

        Catch ex As Exception
            lstDados = Nothing
            Throw ex
        Finally
            objDados = Nothing
        End Try

        Return lstDados

    End Function

    Public Function Add(intFamilaRepresentante As Integer, intFamilaRepresentada As Integer) As Boolean

        Dim blnReturn As Boolean = False
        Dim objDados As Data.Representante = Nothing

        Try

            objDados = New Data.Representante
            blnReturn = objDados.Add(intFamilaRepresentante, intFamilaRepresentada)

        Catch ex As Exception
            blnReturn = False
            Throw ex
        Finally
            objDados = Nothing
        End Try

        Return blnReturn
    End Function
    Public Function Del(intFamilaRepresentante As Integer, intFamilaRepresentada As Integer) As Boolean

        Dim blnReturn As Boolean = False
        Dim objDados As Data.Representante = Nothing

        Try

            objDados = New Data.Representante
            blnReturn = objDados.Del(intFamilaRepresentante, intFamilaRepresentada)

        Catch ex As Exception
            blnReturn = False
            Throw ex
        Finally
            objDados = Nothing
        End Try

        Return blnReturn
    End Function
End Class

File: Jack.Business/Representante.cs
using System;
using System.Collections.Generic;

namespace Jack.Business
{
    public class Representante
    {


        public Representante()
        {
        }
        public List<Model.Familia> ObterMaes(int intFamilia)
        {

            List<Model.Familia> lstDados = null;
            Data.Representante objDados = null;


            try
            {
                objDados = new Data.Representante();
                lstDados = new List<Model.Familia>();
                lstDados = objDados.ObterMaes(intFamilia);

            }
            catch (Exception ex)
            {
                lstDados = null;
                throw ex;
            }
            finally
            {
                objDados = null;
            }

            return lstDados;

        }

        public bool Add(int intFamilaRepresentante, int intFamilaRepresentada)
        {

            bool blnReturn = false;
            Data.Representante objDados = null;


            try
            {
                objDados = new Data.Representante();
                blnReturn = objDados.Add(intFamilaRepresentante, intFamilaRepresentada);

            }
            catch (Exception ex)
            {
                blnReturn = false;
                throw ex;
            }
            finally
            {
                objDados = null;
            }

            return blnReturn;
        }
        public bool Del(int intFamilaRepresentante, int intFamilaRepresentada)
        {

            bool blnReturn = false;
            Data.Representante objDados = null;


            try
            {
                objDados = new Data.Representante();
                blnReturn = objDados.Del(intFamilaRepresentante, intFamilaRepresentada);

            }
            catch (Exception ex)
            {
                blnReturn = false;
                throw ex;
            }
            finally
            {
                objDados = null;
            }

            return blnReturn;
        }
    }
}
